pom.xml

```diff
@@ -128,6 +140,10 @@
       <artifactId>nuxeo-platform-userworkspace-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.nuxeo.ecm.platform</groupId>
+      <artifactId>nuxeo-platform-directory-test</artifactId>
```

Comment:
missing <scope>test</scope> ?

Reply:
You're right.

Reply:
The `test` scope is the default for all `nuxeo-*-test` artifacts. Harmless to explicitly set it though.
```<dependency>
  <groupId>org.nuxeo.ecm.platform</groupId>
  <artifactId>nuxeo-platform-directory-test</artifactId>
  <version>${nuxeo.services.version}</version>
  <scope>test</scope>
</dependency>
```

Reply:
Given the amount of Plugins we have, i feel more confortable in leaving the ```<scope>``` there.
You never know when you refactor something and Plugins starts breaking.